### PR TITLE
Continue button

### LINKED ISSUE #1070: Continue button is not clickable on add-order page
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
Continue button is not clickable when even the quantity select is greater than zero.

**Expected behavior:**
Continue button should be click when quantity is greater than zero.

**Steps to reproduce:**

**Related code:**
https://github.com/fossasia/open-event-frontend/blob/development/app/controllers/events/view/tickets/add-order.js#L8

**Would you like to work on the issue?**

YES

### PR DESCRIPTION
**Pull request by harsh-98**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

sumBy function works for array with  normal javascript  objects. not for ember objects.
So using get() and foreach() to calculate quantity.

Fixes #1070

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-1071-open-event-frontend-fossasia.surge.sh

**Comment by harsh-98** [comment]
@niranjan94 i am not able to register at https://pr-1071-open-event-frontend-fossasia.surge.sh
This is because of https://open-event-api.herokuapp.com/ server failing and producing output 500

**Comment by niranjan94** [comment]
Deployment link: https://pr-1071-open-event-frontend-fossasia.surge.sh

**Comment by harsh-98** [comment]
@geekyd I  am not able to register at https://pr-1071-open-event-frontend-fossasia.surge.sh
This is because of https://open-event-api.herokuapp.com/ server failing and producing output 500

File: app/controllers/events/view/tickets/add-order.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import Ember from 'ember';
-import { sumBy } from 'lodash';
 
 const { Controller, computed } = Ember;
 
 export default Controller.extend({
   hasTicketsInOrder: computed('model.@each.quantity', function() {
-    return sumBy(this.get('model'), 'quantity') > 0;
```
Comment (niranjan94) [inline app/controllers/events/view/tickets/add-order.js]: `sumBy` accepts a custom function. 

```js
sumBy(this.get('model'), item => item.get('quantity')) 
```

File: app/controllers/events/view/tickets/add-order.js [hunk 2]
```diff
@@ -5,7 +5,8 @@ const { Controller, computed } = Ember;
...
+    let sum = sumBy(this.get('model'), item => item.get('quantity'));
+    return sum > 0;
```
Comment (niranjan94) [inline app/controllers/events/view/tickets/add-order.js]: Use the sumBy function here directly. No need to store it.

**Comment by harsh-98** [comment]
@niranjan94 done the requested changes :smile:

**Comment by harsh-98** [comment]
mainly the test are finally because `routes/my-sessions/list.js` doesn't has `AuthenticatedRouteMixin` to checkt whether the used has been authenticated or not.

**Comment by harsh-98** [comment]
@niranjan94  
`return sumBy(this.get('model'), item => item.get('quantity')) > 0;` is giving `get of undefined error`.
but on the other hand,
`    let sum = 0.0;
    this.get('model').forEach(ticket => {
      sum += ticket.get('quantity')
    });
    return sum > 0;` works :thinking:

**Comment by CosmicCoder96** [comment]
Closing due to inactivity.

**Pull request closed without merging**